src/activityRepository.js

```diff
@@ -1,6 +1,11 @@
-class ActivityRepository {
+class ActivtyRepository {
   constructor() {
     this.record = [];
```

Comment:
Most of this stuff looks commented out we may need to just delete it if its dead code